### ISSUE TITLE
provide stealth mode allowing to pause and resume scenario silently

### ISSUE THREAD

**Issue by ThinkBriK** [sloria/doitlive#113]
As described in #44, It would be nice to be able to switch to manual mode via a key combination and switch back to doitlive _silently_. The ctrl+z feature is nice but not quiet enough !

**Comment by sloria** [comment]
Can you clarify what you mean by `ctrl-z` not being quiet enough?

**Comment by ThinkBriK** [comment]
it's still printing the information about the process moving to background
```
$ echo "Greetings"
Greetings
$ echo "This
                                            [1]  + 7055 suspended  doitlive demo -q
                                                                                   %                                                                                                                                                          ~ 
```
I'm using do it live to ensure I don't misstype stuff during presentations but still I need to be able to go back to freetyping to answer questions.

I know it may be ego based but I don't want to pose as an imposter / stackoverflow copy-paster !

**Comment by sloria** [comment]
Ah I see. If you figure out a way to do this, I'd certainly be up for reviewing/merging it.

IMO, it's best to not hide the trick to avoid distracting the audience ("Is he actually typing?"). I like to turn up the typing speed to something ridiculous so that it's clear that I'm not actually typing.

**Comment by sloria** [comment]
This was released in 4.3.0.
